### DISCUSSION TITLE
Online FTP

### DISCUSSION THREAD

**Discussion by z3r01z3r01** [DirectoryLister/DirectoryLister#621] [General]
Hello.! I created a simple script in bash that opens a server in ngrok to be able to access or share files with other people or devices easily as if it were a FTP Online Server.

INSTALLATION
1) Unzip the ZIP file in the root directory of Directory Lister
2) Download Ngrok https://ngrok.com/download
3) Move Ngrok in the root directory of Directory Lister
2) sudo chmod 777 Server.sh && sudo chmod 777 ngrok
3) ./Server.sh or sudo ./Server.sh

The script is commented and free to edit it...

LINK https://www.mediafire.com/file/gms5c3vbk5tx5ir/ServerZIP.zip/file

![Screenshot_2021-05-04_21_42_02](https://user-images.githubusercontent.com/83483229/117086002-bc52c880-ad21-11eb-99eb-c7f37cbc03d3.png)

**Comment by PHLAK** [comment]
Cool, I'm not quite sure how this would be useful but thanks for sharing this. 

However, and I'm sorry I have to ask this, but please don't share unverifiable binary files (i.e. `ngrok`) via this community. I've edited your post to add a warning for now. However, since ngrok is freely available if you want to distribute the shell script and a _link_ to the official ngrok download I'd be happy to remove that warning.
